db/seeds.rb

```diff
@@ -322,46 +322,120 @@
     naf_pledge: 120,
     fund_pledge: 115,
     pledge_sent: true,
-    pledge_generated_at: 70.days.ago,
+    pledge_generated_at: 133.days.ago,
     pledge_generated_by: User.first,
-    updated_at: 81.days.ago,
+    updated_at: 133.days.ago,
   )
   # generate external pledges
   patient.external_pledges.create!(
     source: 'Metallica Abortion Fund',
     amount: 100,
     created_by: user,
-    created_at: 81.days.ago,
+    created_at: 133.days.ago,
   );
   patient.external_pledges.create!(
     source: 'Baltimore Abortion Fund',
     amount: 100,
     created_by: user,
-    created_at: 81.days.ago,
+    created_at: 133.days.ago,
   );
 
   # notes tab
   patient.notes.create!(
     full_text: "Two note, maybe with iffy PII! From the second call.",
     created_by: user2,
-    created_at: 81.days.ago,
+    created_at: 133.days.ago,
   )
 
   # fulfillment
 
   patient.fulfillment.update!(
     fulfilled: true,
-    procedure_date: 60.days.ago,
+    procedure_date: 130.days.ago,
     gestation_at_procedure: "11",
     procedure_cost: 555,
     check_number: 4563,
-    date_of_check: 55.days.ago,
-    updated_at: 51.days.ago,
+    date_of_check: 125.days.ago,
+    updated_at: 125.days.ago,
   )
 
 end
 
 
+# create patients who dropped off
+(1..2).each do |patient_number|
+  # initial create data from voicemail
+  patient = Patient.create!(
+    name: "Archive Dropoff Patient #{patient_number}",
+    primary_phone: "321-9#{patient_number}0-004#{rand(10)}",
```

Comment:
Lightning totally struck and this failed for me because of a primary phone conflict! (it ran fine the second time. Can we mix up the area code on this?